pkg/kubectl/cmd/get.go

```diff
@@ -193,12 +198,26 @@ func RunGet(f *cmdutil.Factory, out io.Writer, cmd *cobra.Command, args []string
 		}
 
 		// print watched changes
-		w, err := r.Watch(rv)
+		// using 0 instead of rv since rv might be outside the watch window
+		w, err := r.Watch("0")
 		if err != nil {
 			return err
 		}
 
 		kubectl.WatchLoop(w, func(e watch.Event) error {
+			accessor, err := meta.Accessor(e.Object)
+			if err != nil {
+				return fmt.Errorf("failed to create metadata accessor: %v", err)
+			}
+			eventrv, err := strconv.Atoi(accessor.GetResourceVersion())
```

Comment:
Shouldn't need to convert to int. Clients should treat resourceVersion as opaque